Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -116,7 +116,7 @@ class GLContextWidget final : public GUI::Frame {
     float m_rotation_speed = 60.f;
```

Comment:
Commit message comment: Can this commit not say Demos+3DFileViewer?

Reply:
Ah, good catch. I had tried a few titles for this one but thought that that prefix wouldn't fit. turns out it would come out at exactly 69 characters.

Reply:
OBE as those demos were removed, this is now a 3DFileViewer only fix :D